code/modules/research/xenobiology/crossbreeding/_status_effects.dm

```diff
@@ -892,11 +892,12 @@
 		if(owner.getFireLoss() > 0)
 			healing_types += BURN
 		if(owner.getToxLoss() > 0)
-			healing_types += TOX
+			owner.adjustToxLoss(-heal_amount, forced = TRUE)
```

Comment:
```suggestion
			healing_types += TOX
```

You've found the correct fix, but applied it to the wrong place. 
This is not where the healing should be taking place.